### PR TITLE
CIV-10580 Judgment by determination docs

### PR DESCRIPTION
**Pull request by bhagyashreesharma90**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-10580

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: build.gradle
Comment (hmcts-version1-ignacio) [inline build.gradle:1]: Remember not to merge this.

File: src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java [hunk 1]
```diff
@@ -247,6 +247,7 @@ public enum CaseEvent {
     SEND_SET_ASIDE_JUDGEMENT_IN_ERROR_LETTER_TO_LIP_DEFENDANT1(CAMUNDA),
     POST_JO_DEFENDANT1_PIN_IN_LETTER(CAMUNDA),
     JUDGMENT_BY_ADMISSION_DEFENDANT1_PIN_IN_LETTER(CAMUNDA),
+    GEN_JUDGMENT_BY_ADMISSION_DOC(CAMUNDA),
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java:250]: Is this event used anywhere?

File: src/main/resources/application.yaml
Comment (hmcts-version1-ignacio) [inline src/main/resources/application.yaml:1]: Why this change?

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by sjamihmcts — DISMISSED** [review]

**Review by Omaira-Melo-Hmcts — DISMISSED** [review]

**Review by sjamihmcts — APPROVED** [review]

**Review by German-A-Amoros — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]
